bedrock/contentful/tests/test_contentful_api.py

```diff
@@ -466,3 +478,171 @@ def test__only_child():
     }
     assert _only_child(node, "text")
     assert not _only_child(node, "dummy-other")
+
+
+# These are what the rich_text_renderer library use for its own tests
+mock_simple_node = {"value": "foo"}
+mock_node = {"content": [{"value": "foo", "nodeType": "text"}]}
+mock_list_node = {"content": [{"content": [{"value": "foo", "nodeType": "text"}], "nodeType": "list-item"}]}
+mock_hyperlink_node = {
+    "data": {"uri": "https://example.com"},
+    "content": [{"value": "Example", "nodeType": "text", "marks": []}],
+}
+
+
+def test__StrongRenderer():
+    assert StrongRenderer().render(mock_simple_node) == "<strong>foo</strong>"
+
+
+def test__EmphasisRenderer():
+    assert EmphasisRenderer().render(mock_simple_node) == "<em>foo</em>"
+
+
+@patch("bedrock.contentful.api.get_current_request")
+def test__LinkRenderer__mozilla_link(mock_get_current_request):
```

Comment:
why double underscores? Just curious.

Reply:
First dunder is a typo (will fix), the second is how I separate the thing being tested from the aspect/case am testing.